lib/src/compile.test.ts

```diff
@@ -0,0 +1,213 @@
+// Copyright 2020 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import {promises as fs} from 'fs';
+import {resolve} from 'path';
+import {fileURLToPath, URL} from 'url';
+
+import {compile, compileString} from './compile';
+import {expectEqualPaths} from '../../spec/helpers/utils';
+import {RawSourceMap, SourceMapConsumer} from 'source-map';
+
+describe('compile', () => {
+  describe('success', () => {
+    describe('input', () => {
+      it('compiles an SCSS string by default', async () => {
+        expect(await compileString({source: 'a {b: c}'})).toBe(
+          'a {\n  b: c;\n}'
+        );
+      });
+
+      // TODO(awjin): compiles an SCSS string explicitly
+      // TODO(awjin): compiles an indented syntax string
+      // TODO(awjin): compiles a plain CSS string
+
+      it('compiles an absolute path', async () => {
+        const path = `${resolve('test.scss')}`;
```

Comment:
I don't think you need the interpolated string here.

```suggestion
        const path = resolve('test.scss');
```